tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -605,7 +609,12 @@ protected function tearDown(): void {
    * @param string $setting
    */
   protected function revertSetting(string $setting): void {
-    \Civi::settings()->set($setting, $this->originalSettings[$setting]);
+    if (isset($this->settings[$setting])) {
```

Comment:
```suggestion
    if (isset($this->originalSettings[$setting])) {
```

Reply:
Doh! Thanks.

Included in the rebased version, which I'll push after a couple more fixes.